modules/gtdbtk/classifywf/meta.yml

```diff
@@ -0,0 +1,78 @@
+name: gtdbtk_classifywf
+description: GTDB-Tk is a software toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes based on the Genome Database Taxonomy GTDB.
+keywords:
+  - GTDB taxonomy
+  - taxonomic classification
+tools:
+  - gtdbtk:
+      description: GTDB-Tk is a software toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes based on the Genome Database Taxonomy GTDB.
+      homepage: https://ecogenomics.github.io/GTDBTk/
+      documentation: https://ecogenomics.github.io/GTDBTk/
+      tool_dev_url: https://github.com/Ecogenomics/GTDBTk
+      doi: "10.1093/bioinformatics/btz848"
+      licence: ['GNU General Public v3 (GPL v3)']
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false,  assembler:'spades' ]
+  - bins:
+      type: The binned fasta files from the assembler
+      description: Fasta files
+      pattern: "*.{fasta,fa}"
+  - database:
+      type: The local copy of the taxonomic database used by GTDB-tk
+      description: The unzipped copy of the database
+      pattern: "*"
+
+output:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - versions:
+      type: file
+      description: File containing software version
+      pattern: "versions.yml"
+  - summary:
+      type: file
+      description: A TSV summary file for the classification
+      pattern: "*.{summary.tsv}"
+  - tree:
+      type: file
+      description: NJ or UPGMA tree in Newick format produced from a multiple sequence alignment
+      pattern: "*.{classify.tree.gz}"
+  - markers:
+      type: file
+      description: A TSV summary file lineage markers used for the classification.
+      pattern: "*.{markers_summary.tsv}"
+  - msa:
+      type: file
+      description: Multiple sequence alignments file.
+      pattern: "*.{msa.fasta.gz}"
+  - user_msa:
+      type: file
+      description: Multiple sequence alignments file for the user-provided files.
+      pattern: "*.{user_msa.fasta.gz}"
+  - filtered:
+      type: file
+      description: A TSV summary of the filtered genomes which were used in the GTDB-tk classification.
```

Comment:
`A list of genomes with an insufficient number of amino acids in MSA.`

https://ecogenomics.github.io/GTDBTk/files/filtered.tsv.html

Reply:
I've update the description ✅

Reply:
Also good catch regarding the gtdb module being sourced from `conda-forge` - thanks Robert!